src/resources/Dissolutions/stepTemplates.ts

```diff
@@ -79,3 +79,20 @@ export const CorpDissolutionSteps = [
     component: ViewComponentNames.DISSOLUTION_REVIEW_CONFIRM
   }
 ]
+
+/**
+ * The Corp step template.
+ * Currently used for BEN, BC, ULC and CCC.
+ */
+// Stepper is not needed for SP and GP ,
+// Need to fix in root to show without stepper later
```

Comment:
Please update these comments. I'd like to see:
- the correct step name in the comments
- the correct entity types this applies to
- a comment that a single step is needed even though the stepper is not used for firm dissolution filings

Reply:
Also please rename this object to match the type of filing this applies to.